Build/Docker.pm

```diff
@@ -20,6 +20,8 @@
 
 package Build::Docker;
 
+use JSON;
```

Comment:
why is SimpleJSON not enough?

If really needed, check how the YAML parser in AppImage parser is loaded optional...

Reply:
SimpleJSON cannot parse json, it can only convert perl data structures to JSON, but I need to convert the .containerinfo files into perl hashes.

I have adjusted the code accordingly.